.circleci/config.yml

```diff
@@ -349,6 +349,30 @@ commands:
           paths:
             - target
 
+  xtask_check_helm:
+    parameters:
+      os:
+        type: string
+    steps:
+      - run:
+          name: Validate helm manifests
+          command: |
+            TODAY=$(date +"%Y-%m-%d")
+
+            CURRENT_KUBE_VERSIONS=$(curl -L https://raw.githubusercontent.com/kubernetes/website/main/data/releases/schedule.yaml \
+              | yq -o json '.' \
+              | jq --raw-output '.schedules[] | select((now | strftime("%Y-%m-%dT00:00:00Z")) as $date | .releaseDate < $date and .endOfLifeDate > $date) | .previousPatches[0].release')
+
+            TEMPLATE_DIR=$(mktemp -d)
+            echo "${CURRENT_KUBE_VERSIONS}" | while read kube_version; do
+              helm template --kube-version "${kube_version}" router helm/chart/router --set autoscaling.enabled=true > "${TEMPLATE_DIR}/router-${kube_version}.yaml"
+
+              kubeval "${TEMPLATE_DIR}/router-${kube_version}.yaml" \
+                --kubernetes-version "${kube_version}" \
+                --strict \
+                --schema-location https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master
+            done
+
```

Comment:
[kubeval](https://github.com/instrumenta/kubeval) is a dead project. Can we use [kubeconform](https://github.com/yannh/kubeconform) (or something still active)?

Reply:
We can and should. I copied this from our internal setup, so hadn't considered different tools.

Quick look and kubeconform looks good. I'll update tomorrow, my time, and push changes back for review once done. Equally happy for author contributions if you get to it before I do. Not going to insult me :)

Reply:
Updated to use kubeconform